client/src/components/generic/Table.js

```diff
@@ -213,7 +214,21 @@ export const Table = ({
   selectedRows = [],
   updateSelectedRows,
   multiSelectAction,
+  filter,
 }) => {
+  const [columnState, setColumnState] = useState(columns);
+  // living checkbox should be hidden if use filters program by HCA
+  // remove checked box if user selects living experience filter and switches to HCA
+  useEffect(() => {
+    if (filter?.programFilter?.value === Program.HCA) {
+      setColumnState((prevColumns) =>
+        prevColumns.filter((i) => i.id !== 'experienceWithMentalHealthOrSubstanceUse')
```

Comment:
Can't we use columns instead of prevColumns?

Reply:
Yeah you're right, some testing I forgot to remove! Will fix

Reply:
I'll update this in my PR.